lib/active_interaction/concerns/runnable.rb

```diff
@@ -59,23 +59,29 @@ def valid?(*)
     # @param (see ClassMethods.run)
     #
     # @return (see #result)
+    #
+    # @raise [Interrupt]
     def compose(other, *args)
       outcome = other.run(*args)
 
-      if outcome.valid?
-        outcome.result
-      else
-        throw :interrupt, outcome.errors
-      end
+      raise Interrupt, outcome.errors unless outcome.valid?
+
+      outcome.result
     end
 
     # @return (see #result=)
     # @return [nil]
+    #
+    # rubocop:disable MethodLength
```

Comment:
If you put this pragma on the same line as `def run` you won't need to re-enable it later.